components/modal/modal.styl

```diff
@@ -34,7 +34,7 @@
 // Container that the modal scrolls within
 .modal
 	overflow: auto
-	overflow-y: scroll
+	overflow-y: auto
```

Comment:
You can remove this line instead. `overflow: auto` will set both x and y to auto

Reply:
yeah

Reply:
This should have been resolved yesterday @nilllzz